src/machine.rs

```diff
@@ -151,8 +151,11 @@ impl<'m> Machine<'m> {
 
         // TODO: Handle fifos. See https://github.com/firecracker-microvm/firecracker-go-sdk/blob/f0a967ef386caec37f6533dce5797038edf8c226/jailer.go#L435
 
-        let mut cmd = Command::new(jailer.jailer_binary().as_os_str());
-        let mut cmd = cmd
+        let mut cmd = &mut Command::new(jailer.jailer_binary().as_os_str());
```

Comment:
Why is `&mut` needed?

Reply:
expected by `arg` and `args`

Reply:
right but you can always borrow it there and that's the conventional way to do it:

```rust
let mut cmd = ...

some_function(&mut cmd);
```

Also in case of methods, it'll deref_mut automatically.